index.html

```diff
@@ -12592,6 +12832,97 @@ <h3>Serializing/Deserializing RDF</h3>
       </dd>
     </dl>
   </section>
+
+  <section class='informative'><h2>The `i18n` Namespace</h2>
+    <p>The `i18n:` namespace is used for describing combinations of <a>language tag</a> and <a>base direction</a> in <a>RDF literals</a>.
+      It is used as an alternative mechanism for describing the [[BCP47]] <a>language tag</a> and <a>base direction</a>
+      of <a>RDF literals</a> that would otherwise be use `xsd:string` or `rdf:langString` datatypes.</p>
+    <p>Datatypes based on this namespace allow round-tripping of JSON-LD documents using base direction,
+      although the mechanism is not otherwise standardized.</p>
+    <p>The <a data-cite="JSON-LD11-API#deserialize-json-ld-to-rdf-algorithm">Deserialize JSON-LD to RDF Algorithm</a>
+      can be used with the <dfn data-cite="JSON-LD11-API#dom-jsonldoptions-rdfDirection">rdfDirection</dfn>
+      set to `datatype` to generate <a>RDF literals</a> using the `i18n` base to create an IRI
+      encoding the <a>base direction</a> along with optional <a>language tag</a> from
+      value objects containing `@direction` by appending to `https://www.w3.org/i18n#`
+      the value of `@language`, if any, followed by an underscore (`"_"`) followed
+      by the value of `@direction`.</p>
```

Comment:
To avoid confusion later, it might be useful to use a language tag with more than one subtag (so the underscore isn't confused with the hyphens used in language tags). In this case, make `ar` -> `ar-EG` (`EG` is the subtag for Egypt), so the serialized form ends up `i18n:ar-EG_rtl`

Reply:
Minor comment: the URL should be `https://www.w3.org/ns/i18n#`, certainly not a URL at the root of the site.

Reply:
Update both the language tag and namespace URI.